sass/sell_media.scss

```diff
@@ -1820,4 +1852,134 @@ input.sell-media-search-text[type=text] {
 			}
 		}
 	}
+}
+
+body .entry-content {
```

Comment:
All of this probably should be remove. Sell Media works with all themes. All themes won't have .entry-content class. So, all of this CSS is probably only specific to the theme you had activated. Also, please use existing CSS classes rather than writing new CSS for positioning. We have many existing layout/grid/positioning classes that can be used that are mobile-responsive also.

Reply:
Yes all removed only used one background color existing property